script/bootstrap

```diff
@@ -15,3 +15,6 @@ if [ -f .env.${NODE_ENV} ]; then
 fi
 
 npm install
+
+# Make sure mocha and coffee are in the PATH
+export PATH="node_modules/.bin:$PATH"
```

Comment:
Oh, [true](https://github.com/afeld/hubot/blob/6dc8db3288f72beb89f93daa399f0eef8f225db4/script/test#L3). Now realizing we can/should probably get rid of [the Makefile](https://github.com/afeld/hubot/blob/6dc8db3288f72beb89f93daa399f0eef8f225db4/Makefile), since the `scripts` are basically doing the same job.